radixdlt/src/main/java/com/radixdlt/epochs/EpochsLocalSyncServiceProcessor.java

```diff
@@ -100,17 +100,13 @@ public void processLocalSyncRequest(LocalSyncRequest request) {
 		if (targetEpoch > currentEpoch.getEpoch()) {
 			log.warn("Request {} is a different epoch from current {} sending epoch sync", request, currentEpoch.getEpoch());
 
-			outsideOfCurrentEpochRequests.compute(targetEpoch, (epoch, list) -> {
-				List<LocalSyncRequest> requests = list == null ? new ArrayList<>() : list;
-				requests.add(request);
-				return requests;
-			});
+			outsideOfCurrentEpochRequests.computeIfAbsent(targetEpoch, epoch -> Lists.newArrayList()).add(request);
```

Comment:
It's rather cosmetic but I prefer to use double underscore parameter name `__` if parameter is not used. So, `epoch -> Lists.newArrayList()` => `__ -> Lists.newArrayList()`. This explicitly shows to the reader unused parameter.

Reply:
Unfortunately that would be a sonarcloud warning right now.  Perhaps we can discuss, and if we want to adopt then we can change sonarcloud rules to suit?